### DISCUSSION TITLE
Single cell transcriptomics

### DISCUSSION THREAD

**Discussion by giscus** [sib-swiss/single-cell-training#44] [Q&A] [answered]
# Single cell transcriptomics

Course on single cell transcriptomics

https://sib-swiss.github.io/single-cell-training/2023.3/

**Comment by MA33RI** [comment]
how to join these upcoming training session plz guide for enrollment.

**Reply by GeertvanGeest** [reply, marked as answer]
You can find more information on single cell courses here: https://www.sib.swiss/training/course/20230703_ISCTR
For SIB courses in general: https://www.sib.swiss/training/upcoming-training-courses

**Comment by Kur1sutaru** [comment]
Thanks for share with us! I want to know if you guys have some email list or slack to connect people working with single cell omics! Thanks a lot 🦄 💻
